metals/src/main/scala/scala/meta/internal/metals/ForwardingMetalsBuildClient.scala

```diff
@@ -52,6 +52,8 @@ final class ForwardingMetalsBuildClient(
   val updatedTreeViews: ju.Set[BuildTargetIdentifier] =
     ConcurrentHashSet.empty[BuildTargetIdentifier]
 
+  def buildHasErrors = !hasReportedError.isEmpty()
```

Comment:
I wonder if we should just check all targets that we depend on when running a particular class :thinking:

Reply:
Probably right? Because if not the same issue will still occur won't it?